examples/attention_sink/example_gqa_sink_bwd_varlen.py

```
@@ -0,0 +1,798 @@
+    cu_seqlens_q: torch.Tensor,
+    cu_seqlens_k: torch.Tensor,
+    max_seqlen_q: int,
+    max_seqlen_k: int,
+    sinks: torch.Tensor,
+    batch_size: int,
+    is_causal: bool,
+    sliding_window: Optional[int] = None,
+    groups: int = 1,
+) -> torch.Tensor:
+    """Reference implementation for varlen attention with sinks."""
+    total_q, num_heads, head_dim = q_unpad.shape
```

Ruff (0.14.10)

Unpacked variable `total_q` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)